mods/ra/weapons/explosions.yaml

```diff
@@ -58,6 +58,13 @@ UnitExplode:
 	Warhead@1Dam: SpreadDamage
 		Falloff: 1000, 368, 135, 50, 18, 7, 0
 
+VisualExplode:
+	Inherits: ^Explosion
+	-Warhead@1Dam: SpreadDamage
+	Warhead@2Eff: CreateEffect
+		Explosions: small_napalm
+		ImpactSounds: firebl3.aud
+
```

Comment:
Does it need  `Inherits: ^Explosion`  and	`-Warhead@1Dam: SpreadDamage` here? I think I was able to do it without.

Reply:
I used the Inherits to
1. create smudge
2. have the valid targets

Reply:
> have the valid targets (EDIT: I don't think we need them)

They're indeed not needed for a purely visual explosion that should trigger unconditionally, but then you should set `VictimScanRadius: 0` to make sure that actors are completely ignored, so that any nearby actors that don't have `Ground` or `Water` target type don't prevent the anim from triggering.

Reply:
@reaperrr: is that a fixup request?